base/loading.jl

```diff
@@ -3170,10 +3185,19 @@ end
         if isempty(modules)
             return true # ignore empty file
         end
-        if ccall(:jl_match_cache_flags, UInt8, (UInt8,), flags) == 0
+        current_flags = ccall(:jl_cache_flags, UInt8, ())
+        if !isempty(tracked_path) 
+            for chi in includes
+                startswith(chi.filename, tracked_path) || continue
+                @debug "Allowing pkgimages=no for $modkey because it falls in coverage/allocation tracking path $tracked_path"
+                current_flags &= 0b11111110 # disable pkgimages flag
```

Comment:
Can we avoid a magic number here? Maybe `CacheFlags(current_flags, pkgimage=no)`

Reply:
`current_flags` needs to be passed into the c func `jl_match_cache_flags` so I didn't think the transform should be done to the julia struct

Reply:
The struct is the only thing in Julia that knows the position of the bits. I am trying to avoid a circumstance where we move the bits around and forget to update this.